inngest/examples/axum/main.rs

```diff
@@ -167,6 +174,39 @@ fn step_run() -> ServableFn<TestData, InngestError> {
     )
 }
 
+fn incorrectly_propagates_error() -> ServableFn<TestData, InngestError> {
```

Comment:
An example is the dev accidentally not propagating the error. At runtime, this will panic. When combined with the async PR, which catches panics, it will be evident to the user either in their logs or in the Inngest telemetry that their function is panicking because they didn't return the control flow error.

Reply:
ah, smart. this will surface on dev, so it doesn't get propagated to prod.

Reply:
yeah will surface on dev assuming the user tests all branches of their code in dev